features/26-mfa/api.md

```diff
@@ -765,6 +765,30 @@ const user = await skygear.auth.mfa.authenticateWithRecoveryCode(recoveryCode);
 
 - POST /mfa/recovery_code/authenticate
 
+## Lost device or changed phone number
+
+### SDK
+
+```typescript
+// Suppose the user has lost their device or changed their phone number.
+// Under default settings, the number of authenticators is 1.
+// So the user should first remove the old authenticator otherwise
+// new registration will never succeed.
+
+// Present an UI to let the user to choose an authenticator to delete.
+const authenticators = await skygear.auth.mfa.getAuthenticators();
```

Comment:
If maximum number of authenticator is one, and that authenticator is lost, the user cannot be authenticated to perform this action?

Reply:
They should authenticate with recovery code.